t/shutdown.t

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Test::More;
+use FindBin qw($Bin);
+use lib "$Bin/lib";
+use MemcachedTest;
+
+my $server;
+my $sock;
+
+# Disabled shutdown (default)
+$server = new_memcached();
+$sock = $server->sock;
+print $sock "shutdown\r\n";
+is(scalar <$sock>, "ERROR: shutdown not enabled\r\n",
+    "error when shutdown is not enabled");
+$server->DESTROY();
```

Comment:
I'm going to edit these out, but fwiw putting the tests in blocks and `my`ing is better. you shouldn't be calling DESTROY(); directly